caffe2/proto/caffe2.proto

```diff
@@ -184,6 +243,34 @@ message OperatorDef {
   // This is an optional string with no assumed characteristics as
   // operators can be constructed in any language.
   optional string debug_info = 10;
+
+  // the domain of the operator
```

Comment:
so it contradicts the above comment about that the namespace is embedded in the name

Reply:
I plan to have a domain here. Will remove the above comment.